operator/hack/common.sh

```diff
@@ -65,6 +65,28 @@ function retry() {
   return 1
 }
 
+function check_version_tag() {
+  local -r version_tag="$1"
+  local -r allow_dirty_tag="$2"
+
+  echo $IMAGE_TAG_BASE
+  if docker manifest inspect "$IMAGE_TAG_BASE:$version_tag" ; then
```

Comment:
What's the purpose of this? In case of remote cluster deployment the existence of an image locally does not mean much, it might be missing in the image repo, leading to image pull errors and failing deployment... 🤔

Reply:
It checks if image is pushed to the registry. `IMAGE_TAG_BASE` equals to `docker.io/stackrox/stackrox-operator` by default. Thus `docker manifest inspect` checks image in the registry, not locally. 
If image is there then it doesn't matter is it a dirty tagged or not so developer will not be bothered to use different command for installing an operator. It's quite fast command and basically just a tiny improvement.

Reply:
You are right, I did not think about such scenario that image is built locally and haven't pushed and installation is triggered. Well, It's was pretty ambiguous check. I removed it